Sources/SwiftProtobuf/ProtoNameResolvers.swift

```diff
@@ -12,22 +12,20 @@
 /// This type contains helper functions to resolve field names based on their
 /// numbers during encoding.
 ///
-/// This type is public since it might be useful to clients who want to write
-/// advanced custom visitors.
-public enum ProtoNameResolvers {
+internal enum ProtoNameResolvers {
```

Comment:
I made this `internal` because changing it to `StaticString` makes it a lot more specialized; I'm not sure if it's really something we want to expose in this form.